src/Errors.ts

```diff
@@ -21,26 +21,38 @@ export class ErrorInfo extends Error {
   }
 }
 
+export const ERR_SPACE_NAME_MISSING = new ErrorInfo({
+  message: 'must have a non-empty name',
```

Comment:
```suggestion
  message: 'must have a non-empty name for the space',
```

Reply:
Updated in 4e4d4f183a63fdba07262cd7f7bb0a0f230aac68